File: data/python/COUNT_NUMBERS_CAN_CONSTRUCTED_USING_TWO_NUMBERS_TRANSLATED.java
====================
public static int f_gold ( int n , int x , int y ) {
  boolean [ ] arr = new boolean [ n + 2 ] ;
  for ( int i = 0 ;
  i < n ;
  i ++ ) arr [ i ] = false ;
  if ( ( x <= n ) && ( y <= n ) ) arr [ x ] = true ;
  if ( ( y <= n ) && ( x >= n ) ) arr [ y ] = true ;
  int result = 0 ;
  for ( int i = Math . min ( x , y ) ;
  i <= n ;
  i ++ ) {
    if ( ( arr [ i ] ) ) {
      if ( ( i + x <= n ) && ( i + x <= n ) ) arr [ i + x ] = true ;
      if ( ( i + y <= n ) && ( i + y <= n ) ) arr [ i + y ] = true ;
      result = result + 1 ;
    }
  }
  return result ;
}

File: data/java/COUNT_NUMBERS_CAN_CONSTRUCTED_USING_TWO_NUMBERS_TRANSLATED.py
import sys

def f_gold ( n , x , y ) :
    arr = [ ]
    if x <= n :
        arr.append ( True )
    if y <= n :
        arr.append ( True )
    result = 0
    for i in range ( min ( x , y ) , n + 1 ) :
        if arr [ i ] :
            if i + x <= n :
                arr.append ( True )
            if i + y <= n :
                arr.append ( True )
            result += 1
    return result

if __name__ == '__main__':
    param = [
    (23,16,16,),
    (56,95,6,),
    (30,63,1,),
    (51,89,46,),
    (21,99,38,),
    (69,63,50,),
    (12,69,73,),
    (44,52,9,),
    (99,65,10,),
    (46,58,37,)
        ]
    n_success = 0
    print("class,mutant,test_index,result")
    for i, parameters_set in enumerate(param):
        try:
            print("COUNT_NUMBERS_CAN_CONSTRUCTED_USING_TWO_NUMBERS," + sys.argv[1] + "," + str(i) + "," + str(f_gold(*parameters_set)))
        except:
            print("COUNT_NUMBERS_CAN_CONSTRUCTED_USING_TWO_NUMBERS," + sys.argv[1] + "," + str(i) + ",EXCEPTION")
